src/pause_menu.py

```diff
@@ -0,0 +1,22 @@
+import curses
+
+class PascalCase:
```

Comment:
Die Klasse sollte nicht `PascalCase` heissen sondern nach dem System benannt werden 😂 Also einfach `PauseMenu` oder so. PascalCase heisst nur, dass Worte in einem Namen durch Grossschreibung getrennt werden und, dass der erste Buchstabe gross ist